cmd/crictl/version.go

```diff
@@ -22,23 +22,20 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
-	"golang.org/x/net/context"
+	internalapi "k8s.io/cri-api/pkg/apis"
 	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
 )
 
-const (
-	criClientVersion = "v1"
-)
+const criClientVersion = "v1"
```

Comment:
since we are using internlapi, maybe use `internalApi.CRIVersionV1`?

This const is meaningless, only recommending for consistency

Reply:
Yes, this makes sense I'll fix it in the next PR.